packages/pigeon/lib/dart_generator.dart

```diff
@@ -599,6 +599,7 @@ if (replyList == null) {
     Root root,
     StringSink sink, {
     required String dartPackageName,
```

Comment:
Is this used anywhere now? Could we adjust the argument instead of adding a second param?